src/components/NumberInput/NumberInput.js

```diff
@@ -88,9 +92,13 @@ NumberInput.propTypes = {
   keepCharPositions: PropTypes.bool,
   autoComplete: PropTypes.string,
   maxLength: PropTypes.number,
+  iconPrefix: PropTypes.string,
```

Comment:
@MayUWish could you use `/** */` to add descriptions for what `iconPrefix` and `iconName` do and what values they are expecting for them to work properly? You can find an example just above for `capitalize` property and `mask` property for `NumberInput`.